README.md

```diff
@@ -32,9 +32,13 @@ pipenv install
 
 ## Usage
 
-- Run `pipenv run python serialize_data.py` to generate JSON file named `endpoints_data.json`. Optionally, you can specify the path of the proposed endpoints docs  using `--file-path` option like:
+- Run `pipenv run python serialize_data.py` to generate JSON file named `endpoints_data.json`. Optionally, you can specify the path of the proposed endpoints docs  using `--file-path` or `-f` option like:
 
-    ```pipenv run python serialize_data.py --file-path /pat/to/endpoints_data.json```
```

Comment:
I guess, the `endpoints_data.json` was wrong in this command earlier?  It needed a `.md` file instead?

Reply:
Yes. The last commit does that.